noxfile.py

```diff
@@ -220,7 +251,7 @@ def pytest_min(session: nox.Session, install_deps: bool = True) -> None:
     if install_deps:
         # install the package itself as editable, so that it is possible to do
         # fast local tests with `nox -R -e pytest_min`.
-        session.install("-e", ".", *PYTEST_DEPS, "-r", "minimum-requirements-ci.txt")
+        session.install("-e", ".", *PYTEST_DEPS, *MIN_DEPS)
```

Comment:
Did you test this works as expected? `install .` installs all the latest dependencies that matches the restrictions at `pyproject.toml`, so the newer versions might be still evaluated. Not sure if by adding the same dependencies afterwards but with the version pinned overrides the lose versions specified in the toml file and picked up by `install .`.